lib/report/index.html.ejs

```diff
@@ -1,742 +1,723 @@
+    }
+
+    .index-item:hover {
+      color: var(--text);
+    }
+
+    .index-node {
+      margin-left: 0.8rem;
+      padding: 1rem;
+    }
+
+    .index-node:last-child {
+      margin-bottom: 1rem;
+    }
+
+    .index-parent {
+      padding: 8px 2px;
+      cursor: pointer;
+    }
+
+    .hidden {
+      display: none;
+    }
+
+    .border-gray {
+      border: 1px solid var(--border);
+    }
 
-<div id="customStats"></div>
+    .border-bottom-gray {
+      border-bottom: 1px solid var(--border);
+    }
 
-<div class="row">
-  <div class="codes-line text-center col-lg-12"></div>
-</div>
-<div class="row">
-  <div class="errors-line text-center col-lg-12"></div>
-</div>
+    span {
+      transition: 0.2s;
+      /* transform-origin: 25% 50%; */
+    }
 
-</div>
+    .chevron-down span {
+      transform: rotate(90deg);
+    }
+  </style>
+</head>
 
-<div class="row">
-  <div class="col-lg-8 col-lg-offset-2">
-    <h3 class="text-center">Raw report data</h3>
-    <p>This is the raw JSON stats output that the report was generated from.</p>
-    <div id="editor">
-    </div>
+<body>
+  <div>
+    <header class="header bg-black-default px-4">
+      <div class="container relative m-auto flex items-center py-3">
+        <!-- logo -->
+        <div style="border-radius: 100%; border: 1px solid var(--border); padding: 2px; margin-right: 8px">
+          <svg width="50" height="50" viewBox="0 0 300 306" fill="white" xmlns="http://www.w3.org/2000/svg">
+            <path
+              d="M177.235 169.156L138.632 132.074L107.952 144.704L93.2929 130.623L228.213 75.9894L180.953 214.829L166.06 200.523L177.235 169.156ZM183.738 150.815L199.254 106.778L156.348 124.505L183.738 150.815Z"
+              fill="white" />
+            <path
+              d="M136.138 135.974L175.036 173.339L208.275 137.452L221.883 150.524L136.914 242.26L123.306 229.189L162.984 186.35L124.087 148.986L84.4087 191.824L70.8005 178.752L155.769 87.0157L169.378 100.088L136.138 135.974Z"
+              fill="white" />
+          </svg>
+        </div>
+        <!-- title and timestamp -->
+        <div>
+          <h1 style="margin-bottom: 4px" id="report-name">
+            Artillery Report
+          </h1>
+          <div id="timestamp"></div>
+        </div>
+
+      </div>
+    </header>
+    <main>
+      <!-- tabs -->
+      <div class="app-heading bg-black-default relative mb-9 pt-4 border-b border-gray-700 px-4">
+        <div class=" container tabs">
+          <div id="charts-tab" class="tab active">Charts</div>
+          <div id="json-tab" class="tab">JSON Output</div>
+        </div>
+      </div>
+
+      <div class="container">
+        <!-- summary blocks -->
+        <div class="app-test-stats w-full">
+          <ul class="flex flex-wrap space-x-3.5" id="summary-blocks">
+          </ul>
+        </div>
+        <div class="row" style="margin-top: 20px; align-items: flex-start;">
+          <div id="charts-col" class="col-lg-10 space-y-4">
```

Comment:
You can replace this with our `<Table>` component from AC. I stripped it down to raw HTML below, would just need to JSify the rows inside `<tbody>`:

```
<div class="relative overflow-x-auto overflow-y-hidden rounded border border-gray-800">
  <table class="w-full border-collapse text-left">
    <thead class="text-gray-default bg-black-200 border-b border-gray-800">
      <tr>
        <th class="whitespace-nowrap p-2">
          Metric
        </th>
        <th class="whitespace-nowrap p-2">
          Value
        </th>
      </tr>
    </thead>
    <tbody class="divide-y divide-gray-800">
      <tr>
        <td class="text-xsm whitespace-nowrap p-2 font-normal text-white">
          http.codes.200
        </td>
        <td class="text-xsm whitespace-nowrap p-2 font-normal text-white">
          78
        </td>
      </tr>
    </tbody>
  </table>
</div>
```

Reply:
Thank you! This fits in very nicely